sway-parse/src/item/item_control_flow.rs

```diff
@@ -0,0 +1,51 @@
+use crate::priv_prelude::*;
```

Comment:
Importing explicitly is a good practice. Is there an issue which prevents this explicit import or is this just lazy coding?

Reply:
Another instance of consistency between all the files in this directory. I didn't really think about this much. Might be worth cleaning up at some point.

Reply:
The point of `priv_prelude` is you just import it all at once.  Pretty much everywhere.  It's a private prelude.